src/protocols/apple-m0110/keyboard_interface.c

```
@@ -71,392 +71,419 @@
- * 
+ *
  * @param data_pin GPIO pin number for DATA line (CLOCK will be data_pin + 1)
  *                 Must be even number to allow consecutive pin allocation
  *                 Recommended: Use pins with good signal integrity (avoid LED pins)
- * 
+ *
  * @note Call this function once during system initialization
  * @note Requires keyboard_interface_task() to be called regularly for operation
  * @see keyboard_interface_task() for ongoing protocol management
  */
 void keyboard_interface_setup(uint data_pin) {
```

Cppcheck (2.19.0)

[style] The function 'keyboard_interface_setup' is never used.

(unusedFunction)